plugins/importexport/native/filter/NativeXmlSubmissionFilter.php

```diff
@@ -238,4 +243,22 @@ public function getImportFilter($elementName)
     {
         assert(false); // Subclasses should override
     }
+
+    /**
+     * @param string $stageName
+     * @param Submission|null $submission
+     * @return void
+     * @throws \Exception
+     */
```

Comment:
Maybe to remove these, but add the description for the function.

Because we actually do not handle all submission stages here, we could also do the check 
`if ($stageName === "externalReview" or $stageName === "internalReview") {`
above in the `handleElement` function and call this function `createReviewRound`. But this is just optional, as you think/would like...

Reply:
@bozana yes, that makes sense!  I changed the signature of the function.